contrib/monitored_resource_util/src/main/java/io/opencensus/contrib/monitoredresource/util/LabelKey.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.monitoredresource.util;
+
+/**
+ * {@link LabelKey} represents the associated property of a {@link MonitoredResource}.
+ *
+ * @since 0.13
+ */
+public enum LabelKey {
```

Comment:
Why public? Maybe this should be in MonitoredResource.

Reply:
Because this is part of the returned value of labels. If we don't make it public, we would need to return a `Map<String, String>` as labels.

In addition, I would prefer to leave it as a top-level class, similar to `TagKey` vs `TagContext`.